src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/email/TaskEmail.java

```diff
@@ -0,0 +1,40 @@
+package org.fogbowcloud.saps.engine.core.dispatcher.email;
+
+import com.google.gson.annotations.SerializedName;
+import org.fogbowcloud.saps.engine.core.archiver.storage.AccessLink;
+import org.fogbowcloud.saps.engine.core.model.SapsImage;
+
+import java.util.Date;
+import java.util.List;
+
+public class TaskEmail {
```

Comment:
Now, I realize this is not a good name. Maybe, TaskCompleteInfo?

Reply:
TaskFinishedInfo

Reply:
Done